source/Calamari.Common/FeatureToggles/FeatureToggle.cs

```diff
@@ -8,7 +8,6 @@
     public enum FeatureToggle {
         SkunkworksFeatureToggle,
         OidcAccountsFeatureToggle,
-        KubernetesLiveObjectStatusFeatureToggle,
```

Comment:
this can also be removed as we now _always_ report manifests, no matter if LOS is enabled or not